CHANGELOG.md

```diff
@@ -10,18 +10,24 @@ internal API changes are not present.
 Main (unreleased)
 -----------------
 
+> **BREAKING CHANGES**: This release has breaking changes. Please read entries
+> carefully and consult the [upgrade guide](https://grafana.com/docs/agent/latest/upgrade-guide/)
+> for specific instructions.
```

Comment:
You don't have to give the link, `[upgrade guide]` is defined at the bottom of the CHANGELOG. 

```suggestion
> **BREAKING CHANGES**: This release has breaking changes. Please read entries
> carefully and consult the [upgrade guide][] for specific instructions.
```